linear-base.cabal

```diff
@@ -161,6 +161,7 @@ library examples
         storable-tuple,
         vector,
         text
+    default-language: Haskell2010
```

Comment:
This is a fix for Hackage. More recent Cabal versions apparently let you factor the `default-language` into one place. But its a bigger change, I don't want to do it for this PR.

That being said, I don't even want to upload this. Looking stuff up now.

Reply:
Actually, I guess it's fine, you always upload your examples and stuff.